ENIGMAsystem/SHELL/Graphics_Systems/General/GSscreen.cpp

```diff
@@ -228,6 +230,13 @@ static inline void draw_gui()
   d3d_set_zwriteenable(zwrite);
 }
 
+static inline void draw_gui_after_hooks() {
+  // if any extension wants to draw over the user
+  // GUI let's go ahead and call those hooks now
+  for (auto draw_gui_after_hook : extension_draw_gui_after_hooks)
+    draw_gui_after_hook();
```

Comment:
@JoshDreamland See, this is why it would be cool if extensions could just hook into, before, or after any arbitrary user event without us having to add more hooks. That's why I'm very interested to see #997 in its fullest form some time soon.